### PR TITLE
[module_manager] remove extra padding on selectelement for dynamictable

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

Resolves the unnecessary padding to SelectElement components used inside the DynamicTable component.

#### Testing instructions (if applicable)

1. Visit the module_manager module.
2. Observe the fix for the UI carousel scrollers.

#### Link(s) to related issue(s)

* Resolves #  (Reference the issue this fixes, if any.)
#6697

### CODE REVIEW

**Comment by ridz1208** [comment]
@zaliqarosli can you give this one a quick look ?

**Comment by maltheism** [comment]
@zaliqarosli thanks that makes way more sense. Let me know if I missed something with latest commit.

File: jsx/Form.js [hunk 1]
```diff
@@ -477,15 +477,15 @@ class SelectElement extends Component {
...
+    let inputClass = this.props.noMargins ? '' : 'col-sm-12';
     if (this.props.label && this.props.label != '') {
       label = (
         <label className="col-sm-3 control-label" htmlFor={this.props.label}>
           {this.props.label}
           {requiredHTML}
         </label>
       );
-      inputClass = 'col-sm-9';
+      inputClass = this.props.noMargins ? '' : 'col-sm-9';
```
Comment (zaliqarosli) [inline jsx/Form.js:488]: i would suggest leaving this line unchanged. i think the select element looks better with input classes col-sm-3 and col-sm-9 if it has a label

Screen Shot 2020-06-12 at 2 12 14 PM

Comment (zaliqarosli) [reply jsx/Form.js:488]: the last two rows is with line 488 untouched. what do you think @maltheism ?

Comment (maltheism) [reply jsx/Form.js:488]: @zaliqarosli yah seems good. I'll change it back.

**Comment by maltheism** [comment]
Thanks @zaliqarosli! Latest commit has the new changes.

**Pull request merged**